### ISSUE TITLE
WCAG 1.4.3: Elements must have sufficient color contrast (.active)

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2276]
#### Issue
`Elements must have sufficient color contrast` ([color-contrast](https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI))
#### Target application
[Microsoft MakeCode for micro:bit](https://makecode.microbit.org/#editor)
#### Element path
.filemenu > .active
#### Snippet
`<a tabindex="0" role="treeitem" aria-selected="true" aria-label="main.ts, it is the current opened file in the JavaScript editor" class="active item">main.ts <i class="icon align left " aria-hidden="true" role="presentation"></i></a>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 3.36 (foreground color: #ffffff, background color: #7187df, font size: 10.3pt, font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 75.0.3770.142
This accessibility issue was found using Accessibility Insights for Web 2.7.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

![Screen Shot 2019-07-17 at 5 25 28 PM](https://user-images.githubusercontent.com/5615930/61420550-e373a800-a8b7-11e9-8b99-3c4ee38f0814.png)

**Comment by jwunderl** [comment]
Repros even in high contrast mode:

<img width="410" alt="Screen Shot 2019-08-02 at 2 21 48 PM" src="https://user-images.githubusercontent.com/5615930/62399547-fed5e880-b530-11e9-938a-d7f691059bb9.png">
